pom.xml

```diff
@@ -452,7 +452,7 @@
     <version.junit>4.11</version.junit>
     <version.org.jboss.byteman>2.1.3</version.org.jboss.byteman>
     <version.org.jboss.arquillian.core>1.1.2.Final-wildfly-1</version.org.jboss.arquillian.core>
-    <version.org.jboss.jboss-transaction-spi>7.1.0.Final</version.org.jboss.jboss-transaction-spi>
+    <version.org.jboss.jboss-transaction-spi>7.1.1.Final-SNAPSHOT</version.org.jboss.jboss-transaction-spi>
```

Comment:
Yes that's right, we will need to release the SPI before merging this PR.

Also, I hope you don't mind but I squashed all the commits into the original one so the resulting commit will be made in your name because I originally clone the branch for this work from one of yours (I guess as long as we are happy that the quality is good it does not matter whose name goes on the commit).